straxen/contexts.py

```diff
@@ -2,6 +2,14 @@
 import strax
 import straxen
 
+# list of contexts that gets tracked in runDB context collection
+# needs to be maintained for each straxen release
+context_list = ['xenonnt_led', 
+                'xenonnt_online',
+                'xenonnt_initial_commissioning',
+                'xenonnt_simulation'
```

Comment:
Not sure if you want to track this at the moment as it's more of a placeholder which is adapted at the moment by people wanting to simulate stuff. 

This of course may change if we are going to make a centralized simulation context with also shared files, but at the moment this is not used as such.

Reply:
You're talking about the simulation one specifically right? If so I will remove. I don't want to be in charge of maintaining this list, just was proposing the existence of one

Reply:
Indeed, I was only mentioning `xenonnt_simulation`

Reply:
Done